packages/@glimmer/test-helpers/test/i-n-u-r-test.ts

```diff
@@ -13,8 +14,8 @@ QUnit.test('Can set properties', assert => {
 });
 
 QUnit.test('Can take basic snapshots', assert => {
-  let div = document.createElement('div');
-  let text = document.createTextNode('Foo');
+  let div = document.createElement('div') as Simple.Element;
```

Comment:
can we type the doc variable set to document as Simple.Document?

Reply:
Good call.